tests/unit/data/test_extract_metadata.py

```diff
@@ -305,14 +305,14 @@ def test_containing_polygon_overlap_within_separate(cube_1, cube_2, cube_3, cube
     @staticmethod
     def test_aircraft_cube(aircraft_cube):
         """
-        GIVEN a single cube
+        GIVEN a cubelist of aircraft data
```

Comment:
I feel that we should have a unit test of both of these instances; one using a single cube and one using a cubelist.  If we don't have the sample data of this available right now then it is best to write both tests and comment out the one without sample data (with a comment about why this is the case) and write a Jira ticket to reinstate tests once data is available.  Also write a Jira ticket to obtain or create the missing data.  Or put both tasks on same ticket.  Just as long as we don't forget why a test is commented out.

Reply:
The two tests below this cover both of these instances (the function working for a cube and a cubelist); the aim of this test is to ensure it works with the some real flight data. 
Here I've just updated the description after I changed the sample data to cover more than one variable, for the multiplots in the notebook (but is also probably more accurate for testing, as there are no aircraft files that are single cubes).

Reply:
Well then I apologise.  Nice work.